src/Bicep.LangServer/Handlers/BicepBuildCommandHandler.cs

```diff
@@ -77,7 +80,7 @@ private string GenerateCompiledFileAndReturnBuildOutputMessage(string bicepFileP
             if (context is null)
             {
                 SourceFileGrouping sourceFileGrouping = SourceFileGroupingBuilder.Build(this.fileResolver, this.moduleDispatcher, new Workspace(), documentUri.ToUri());
-                compilation = new Compilation(AzResourceTypeProvider.CreateWithAzTypes(), sourceFileGrouping, null);
+                compilation = new Compilation(AzResourceTypeProvider.CreateWithAzTypes(), sourceFileGrouping, this.configurationManager.GetBuiltInConfiguration());
```

Comment:
Shouldn't we attempt to look up the configuration for the file path here?

Reply:
I was basically replacing `null` with `GetBuiltInConfiguration()` using global search, but this is a good catch. Will change it to load configuration for the documentUri.